packages/base-driver/lib/protocol/routes.js

```diff
@@ -377,10 +369,6 @@ export const METHOD_MAP = /** @type {const} */ ({
       deprecated: true,
     },
   },
-  '/session/:sessionId/location': {
-    GET: {command: 'getGeoLocation'},
-    POST: {command: 'setGeoLocation', payloadParams: {required: ['location']}},
-  },
```

Comment:
Moved to Appium only section

Reply:
you may use the `// #region X ... // #endregion` comments to create collapsible regions in various editors, like VSCode.